docs/pages/distribution/optimizing-updates.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Optimizing Updates
+---
+
+## Table of contents
```

Comment:
```suggestion
## Table of contents
```

we should have a section before this that introduces some concepts like asset bundling and which assets are included when you run `expo publish`. we should probably also provide a tool that lets people see their current estimated app size by getting the size of all of the assets and estimating js bundle size

Reply:
We don't have a tool for this but I will suggest exporting the update and estimating from there.